### PR TITLE
Convert README.md from Eclipse to IntelliJ

### LINKED ISSUE #81: Convert docs to use Intellij
Current docs are written for Eclipse. Convert them to use Intellij.

### PR DESCRIPTION
**Pull request by limmlingg**
Part of #81 (does not cover the Eclipse tutorial slides in the docs folder)

### CODE REVIEW

**Comment by limmlingg** [comment]
Not sure if I should modify the parts in Learning Outcomes since prof Damith intends to shorten it.

File: README.md [hunk 1]
```diff
@@ -40,19 +40,19 @@ Please refer to the [Setting up](#setting-up) section to learn how to set up the
...
 ## Starting the program
 
-**Using Eclipse**
+**Using IntelliJ**
 
-1. Find the project in the `Project Explorer` or `Package Explorer` (usually located at the left side)
-2. Right click on the project
-3. Click `Run As` > `Java Application`
+1. Find the project in the `Project Explorer` (usually located at the left side)
+2. Go to the `src` folder and locate the `AddressBook` file.
```
Comment (yamgent) [inline README.md]: 1a. If the `Project Explorer` is not visible, press <kbd>Alt</kbd>+<kbd>1</kbd> for Windows/Linux, <kbd>Cmd</kbd>+<kbd>1</kbd> for macOS.

File: README.md [hunk 2]
```diff
@@ -144,15 +144,21 @@ Example:
...
-3. Click `General` > `Existing Projects into Workspace` > `Next`
-4. Click `Browse`, then locate the project's directory
-5. Click `Finish`
+* IntelliJ IDE
+
+**Importing the project into IntelliJ**
+
+1. Open IntelliJ (if you are not in the welcome screen, click `File` > `Close Project` to close the existing project dialog first)
+2. Click `File` > `New` > `Project from Existing Sources...`
+3. Set up the correct JDK version for Gradle
```
Comment (yamgent) [inline README.md]: The instruction seems to be wrong, because we do not use Gradle at level 1.

Comment (Zhiyuan-Amos) [inline README.md]: Step 2 is wrong as well. :<

**Comment by CanIHasReview** [comment]
# v1

@limmlingg submitted v1 for review.

* [v1 1/1] [Convert README\.md from Eclipse to IntelliJ](https://github.com/se-edu/addressbook-level1/pull/85/commits/6d2cabd4caeefa3d7d62c6edd978efd22775a883)

([:books: Archive](https://github.com/se-edu/addressbook-level1/compare/a4515e3895ca1af1fc7c1753dec0a90c37366032...6d2cabd4caeefa3d7d62c6edd978efd22775a883))
<details>
<summary>Checkout this PR version locally</summary>

```sh
git fetch https://github.com/se-edu/addressbook-level1.git refs/pr/85/1/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you wish to fetch this PR to.

</details>

File: README.md [hunk 3]
```diff
@@ -40,19 +40,20 @@ Please refer to the [Setting up](#setting-up) section to learn how to set up the
...
 ## Starting the program
 
-**Using Eclipse**
+**Using IntelliJ**
 
-1. Find the project in the `Project Explorer` or `Package Explorer` (usually located at the left side)
-2. Right click on the project
-3. Click `Run As` > `Java Application`
+1. Find the project in the `Project Explorer` (usually located at the left side)
+   1. If the `Project Explorer` is not visible, press <kbd>ALT</kbd>+<kbd>1</kbd> for Windows/Linux, <kbd>CMD</kbd>+<kbd>1</kbd> for macOS.
```
Comment (Zhiyuan-Amos) [inline README.md]: press... to open the `Project Explorer` tab.

File: README.md [hunk 4]
```diff
@@ -144,15 +145,20 @@ Example:
...
-2. Click `File` > `Import`
-3. Click `General` > `Existing Projects into Workspace` > `Next`
-4. Click `Browse`, then locate the project's directory
-5. Click `Finish`
+* IntelliJ IDE
+
+**Importing the project into IntelliJ**
+
+1. Open IntelliJ (if you are not in the welcome screen, click `File` > `Close Project` to close the existing project dialog first)
+3. Set up the correct JDK version
```
Comment (Zhiyuan-Amos) [inline README.md]: 2, not 3 :P

Comment (yamgent) [inline README.md]: Actually, we can put all `1.` and markdown will automatically number properly, but I don't think this PR should be concerned about that.

File: README.md [hunk 5]
```diff
@@ -40,19 +40,20 @@ Please refer to the [Setting up](#setting-up) section to learn how to set up the
...
-**Using Eclipse**
+**Using IntelliJ**
 
-1. Find the project in the `Project Explorer` or `Package Explorer` (usually located at the left side)
-2. Right click on the project
-3. Click `Run As` > `Java Application`
+1. Find the project in the `Project Explorer` (usually located at the left side)
+   1. If the `Project Explorer` is not visible, press <kbd>ALT</kbd>+<kbd>1</kbd> for Windows/Linux, <kbd>CMD</kbd>+<kbd>1</kbd> for macOS to open the `Project Explorer` tab.
+2. Go to the `src` folder and locate the `AddressBook` file.
+3. Right click the file and select `Run AddressBook.main()`
```
Comment (yamgent) [inline README.md]: The full stops are not really consistent (sometimes have, sometimes don't have).

File: README.md [hunk 6]
```diff
@@ -144,15 +145,20 @@ Example:
...
+1. Open IntelliJ (if you are not in the welcome screen, click `File` > `Close Project` to close the existing project dialog first)
+2. Set up the correct JDK version
+   1. Click `Configure` > `Project Defaults` > `Project Structure`
+   2. Click `New...` and find the directory of the JDK
+3. Click `Import Project`
+4. Locate the project directory and click `OK`
+5. Select `Create project from existing sources` and click `Next`
+6. Rename the project if you want. Click `Next`.
+7. Ensure that your src folder is checked. Keep clicking `Next`.
+8. Click `Finish`
```
Comment (yamgent) [inline README.md]: Same thing with this list (full stop not consistent).

**Comment by CanIHasReview** [comment]
# v2

@limmlingg submitted v2 for review.

* [v2 1/1] [Convert README\.md from Eclipse to IntelliJ](https://github.com/se-edu/addressbook-level1/pull/85/commits/a48fab4edb6cc9a80963615accf4984ce1ecba74)

([:books: Archive](https://github.com/se-edu/addressbook-level1/compare/a4515e3895ca1af1fc7c1753dec0a90c37366032...a48fab4edb6cc9a80963615accf4984ce1ecba74)) ([:chart_with_upwards_trend: Interdiff between v1 and v2](https://github.com/se-edu/addressbook-level1/compare/5f09f35d60db7f41043f8b394d3d0658fe137c90~1...5f09f35d60db7f41043f8b394d3d0658fe137c90))
<details>
<summary>Checkout this PR version locally</summary>

```sh
git fetch https://github.com/se-edu/addressbook-level1.git refs/pr/85/2/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you wish to fetch this PR to.

</details>

File: README.md [hunk 7]
```diff
@@ -202,28 +208,27 @@ corresponding exercises.
...
+* Learn [how to set up a new project in IntelliJ (from jetbrains.com)]
+  (https://www.jetbrains.com/help/idea/creating-running-and-packaging-your-first-java-application.html). <br>
+  Create a new project in IntelliJ and write a small HelloWorld program.
 * Download the source code for this project: Click on the `clone or download` link above and either,
    1. download as a zip file and unzip content.
    2. clone the repo (if you know how to use Git) to your Computer.
-* [Set up](#setting-up) the project in Eclipse.
-* [Run the program](#starting-the-program) from within Eclipse, and try the features described in
+* [Set up](#setting-up) the project in IntelliJ.
+* [Run the program](#starting-the-program) from within IntelliJ, and try the features described in
```
Comment (Zhiyuan-Amos) [inline README.md]: missing period

File: README.md [hunk 8]
```diff
@@ -251,6 +254,8 @@ Here are some things you can do in your demonstration:
 4. 'Step into', and 'step out of', methods as you step through the code
 5. ...
 
+Resources: [Debugger Basics (from jetbrains.com)](https://www.jetbrains.com/help/idea/debugger-basics.html)
```
Comment (Zhiyuan-Amos) [inline README.md]: do we need to put the parentheses `from jetbrains.com`? seems a bit random

Comment (limmlingg) [inline README.md]: Most of the `Resources` have it

File: README.md [hunk 9]
```diff
@@ -202,28 +208,27 @@ corresponding exercises.
...
 ## Set up a project in an IDE `[LO-IdeSetup]`
 
-#### Exercise: Setup project in Eclipse 
+#### Exercise: Setup project in IntelliJ 
 
-* Learn [how to set up a new project in Eclipse]
-  (https://se-edu.github.io/addressbook-level1/doc/Getting started with Eclipse.pptx). <br>
-  Create a new project in Eclipse and write a small HelloWorld program.
+* Learn [how to set up a new project in IntelliJ (from jetbrains.com)]
+  (https://www.jetbrains.com/help/idea/creating-running-and-packaging-your-first-java-application.html). <br>
```
Comment (Zhiyuan-Amos) [inline README.md]: there seems to be an unintended error here (it was present in the original file as well), but I think we can fix it ba. Seems weird to have square brackets at `[how to set up a new project in IntelliJ (from jetbrains.com)]`

File: README.md [hunk 10]
```diff
@@ -202,28 +208,27 @@ corresponding exercises.
...
 The `AddressBook.java` code is rather long, which makes it cumbersome to navigate by scrolling alone. 
 Navigating code using IDE shortcuts is a more efficient option. 
-For example, <kbd>F3</kbd> will navigate to the definition of the method/field at the cursor.
+For example, <kbd>CTRL</kbd>+<kbd>B</kbd> will navigate to the definition of the method/field at the cursor.
  
 #### Exercise: Learn to navigate code using shortcuts
 
-Learn some Eclipse code navigation shortcuts 
-(you can use Web resources like [this one](https://www.shortcutworld.com/en/win/Eclipse.html)). 
+Learn some IntelliJ code navigation shortcuts
```
Comment (yamgent) [inline README.md]: Missing full stop.

**Comment by CanIHasReview** [comment]
# v3

@limmlingg submitted v3 for review.

* [v3 1/1] [Convert README\.md from Eclipse to IntelliJ](https://github.com/se-edu/addressbook-level1/pull/85/commits/a185f4c6490492b7ee5ace24888b669e2aa7286e)

([:books: Archive](https://github.com/se-edu/addressbook-level1/compare/a4515e3895ca1af1fc7c1753dec0a90c37366032...a185f4c6490492b7ee5ace24888b669e2aa7286e)) ([:chart_with_upwards_trend: Interdiff between v2 and v3](https://github.com/se-edu/addressbook-level1/compare/14e44531d6d524d058aa92308f183928d788bdbf~1...14e44531d6d524d058aa92308f183928d788bdbf))
<details>
<summary>Checkout this PR version locally</summary>

```sh
git fetch https://github.com/se-edu/addressbook-level1.git refs/pr/85/3/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you wish to fetch this PR to.

</details>

**Comment by damithc** [comment]
Did we forget to delete the unused pptx files? 😛

**Pull request merged**